deploy/db/mysql.yaml

```diff
@@ -73,7 +73,7 @@ spec:
         - name: MYSQL_ROOT_PASSWORD
           valueFrom:
             secretKeyRef:
-              key: password.txt
+              key: password
```

Comment:
Where is the password file?

Reply:
now just set the value, not use a password file.